src/structures/GuildEmoji.js

```diff
@@ -10,14 +10,15 @@ const Permissions = require('../util/Permissions');
  * @extends {BaseGuildEmoji}
  */
 class GuildEmoji extends BaseGuildEmoji {
-  /**
```

Comment:
The constructor should stay documented.
Just dropping the `@name`, `@kind`, and `@memberof` should work.